client/src/components/Modal.jsx

```diff
@@ -1,28 +1,85 @@
 import React from 'react';
+import moment from 'moment';
+import _ from 'underscore';
+import Item from './Item.jsx';
 import Review from './Review.jsx';
 import ReviewContainer from './ReviewContainer.jsx';
-import styles from './css/modalStyles.css.js';
+import modalStyles from './css/modalStyles.css.js';
 
 const Modal = (props) => {
   if (!props.showModal) {
     return null;
   } else {
     return (
-      <div style={styles.modal} onClick={props.onClose}>
-        <section style={styles.modalMain}>
-          {/* <div style={styles.reviewContent}> */}
-          <img style={styles.image} src="https://i.etsystatic.com/iap/c08960/1735567682/iap_640x640.1735567682_9rntz5h8.jpg?version=0"></img>
-          <ReviewContainer showPrice="true"/>
-          {/* </div> */}
-          {/* <div> */}
-          <button style={styles.button} onClick={props.onClose}>
-            <img style={styles.buttonImg}src="https://cdn0.iconfinder.com/data/icons/elasto-online-store/26/00-ELASTOFONT-STORE-READY_close-512.png"></img>
+      <div style={modalStyles.modal} onClick={props.onClose}>
+        <section style={modalStyles.modalMain}>
+          <img style={modalStyles.image} src={reviews[0].item_img}></img>
+          <div>
+            <div style={modalStyles.userInfo}>
+              <img style ={modalStyles.profileImg} src={reviews[0].user_img}></img>
+              <div style={modalStyles.username}>{reviews[0].user_name}</div>
+              <div style={modalStyles.date}>{moment(reviews[0].created_at).format('MMM DD, YYYY')}</div>
+            </div>
+            <div style={modalStyles.review}>
+              <div>
+                {_.times(reviews[0].stars, (n) =>{
+                  return (
+                    <img style={modalStyles.star} src="https://s3-us-west-1.amazonaws.com/anstyicons/icon-star-512.png"></img>
```

Comment:
This isn't a helpful comment, but you can also use .svg stars here and use border/fill to color them.  I would say dont change them at this point :)

Reply:
thanks! yeah I am definitely getting to the point where I need to weigh my options, this seemed quickest and easiest given time contraints